File: KoKoslan/src/java/ast/KoKoNum.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.java.ast;
import kokoslan.java.eval.*;

public class KoKoNum extends KoKoAtom<Double>{
   public KoKoNum(Double value){
      super(value);
   }
   @Override
   public KoKoValue eval(KoKoContext ctx){
	   return new KoKoNumValue(getValue());
	   
   }
   
}

File: KoKoslan/src/kotlin/ast/KoKoNum.kt
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.kt.ast

import kokoslan.kt.eval.*
import kokoslan.kt.exception.*

class KoKoNum(value : Double) : KoKoAtom<Double>(value){

   override fun eval(ctx : KoKoContext) : KoKoValue {
	   return KoKoNumValue(getValue())
   }
   
}